pools/categories/index.json

```
@@ -63,6 +63,14 @@
     "description": "Vote Escrow Pool Tokens",
     "file": "ve8020.json"
   },
+  {
+    "id": "mode",
+    "name": "Mode Points",
+    "description": "LPs in this pool earn 5x Mode points on the TVL of the pool (Mode and USDC) and earn a weekly share of $OP when the BPT is staked on the Mode dashboard for an additional point multiplier.",
+    "url": "https://app.mode.network/early/",
+    "file": "mode.json",
+    "fileIcon": "mode.svg"
+  },
```

Please don't include specific multiples here as they can change and this is static metadata. 
```suggestion
  {
    "id": "points_mode",
    "name": "Mode Points",
    "description": "LPs in this pool earn Mode points on the TVL of the pool (Mode and USDC) and earn a weekly share of $OP when the BPT is staked on the Mode dashboard for an additional point multiplier.",
    "url": "https://app.mode.network/early/",
    "file": "points_mode.json",
    "fileIcon": "points_mode.svg"
  },
```